### PR TITLE
0.9.0 changes

### PR DESCRIPTION
**Pull request by DomPeliniAerospike**

### CODE REVIEW

File: src/aerospike_vector_search/shared/conversions.py [hunk 1]
```diff
@@ -90,6 +90,46 @@ def fromVectorDbNeighbor(input: types_pb2.Neighbor) -> types.Neighbor:
...
+        # sets=input.setFilter
+        hnsw_params=types.HnswParams(
+            m=input.hnswParams.m,
+            ef_construction=input.hnswParams.efConstruction,
+            ef=input.hnswParams.ef,
+            batching_params=types.HnswBatchingParams(
+                max_records=input.hnswParams.batchingParams.maxRecords,
+                interval=input.hnswParams.batchingParams.interval,
+            ),
+            # caching_params=types.HnswCachingParams(
```
Comment (jdogmcsteezy) [inline src/aerospike_vector_search/shared/conversions.py:111]: Should these be commented out?

File: src/aerospike_vector_search/shared/conversions.py [hunk 2]
```diff
@@ -90,6 +90,46 @@ def fromVectorDbNeighbor(input: types_pb2.Neighbor) -> types.Neighbor:
     )
 
 
+def fromIndexDefintion(input: types_pb2.IndexDefinition) -> types.Neighbor:
```
Comment (jdogmcsteezy) [inline src/aerospike_vector_search/shared/conversions.py:93]:
```suggestion
def fromIndexDefintion(input: types_pb2.IndexDefinition) -> types.IndexDefinition:
```

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]

File: src/aerospike_vector_search/shared/conversions.py (hunk 1 above)
Comment (DomPeliniAerospike) [reply src/aerospike_vector_search/shared/conversions.py:111]: Leaving in for a future improvement.

File: src/aerospike_vector_search/shared/conversions.py (hunk 2 above)
Comment (DomPeliniAerospike) [reply src/aerospike_vector_search/shared/conversions.py:93]: I will commit this direct to stage to skip CI run.

**Pull request merged**